spec/defines/download_file_spec.rb

```diff
@@ -229,4 +229,20 @@
       }
     end
   end
+
+  describe 'the proxyAddress parameter' do
+    let(:title)  { 'Download nodejs installer' }
+    let(:params) {{
+        :url => 'http://my.server/test.exe',
+        :destination_directory => 'c:\temp',
+        :destination_file => 'foo.exe',
+        :proxyAddress => 'http://localhost:9090'
+    }}
+
+    describe 'is still supported' do
```

Comment:
Right, we cannot easily test for the invocation of the `warning` function. I guess this test on its own is litle fun...I'd be fine with dropping it. It will have to go as soon as the parameter is actually removed, anyway.